app/src/pages/common/uiExtensionPage/uiExtensionPage.jsx

```diff
@@ -14,24 +14,37 @@
  *  limitations under the License.
  */
 
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
-import { PageLayout, PageSection, PageHeader } from 'layouts/pageLayout';
+import { PageSection } from 'layouts/pageLayout';
 import { ExtensionLoader, extensionType } from 'components/extensionLoader';
+import { Header } from 'pages/inside/projectSettingsPageContainer/header';
+import classNames from 'classnames/bind';
+import styles from './uiExtensionPage.scss';
+
+const cx = classNames.bind(styles);
 
 export const UiExtensionPage = ({ extensions, activePluginPage }) => {
   const extension = React.useMemo(() => extensions.find((ex) => ex.name === activePluginPage), [
     extensions,
     activePluginPage,
   ]);
 
+  const [headerNodes, setHeaderNodes] = useState({});
+
   return (
-    <PageLayout>
-      {extension && <PageHeader breadcrumbs={[{ title: extension.title || extension.name }]} />}
+    <>
+      {extension && (
+        <div className={cx('header')}>
+          <Header titleNode={headerNodes.titleNode} title={extension.title || extension.name}>
+            {headerNodes.children}
+          </Header>
+        </div>
+      )}
       <PageSection>
-        <ExtensionLoader extension={extension} withPreloader />
+        <ExtensionLoader extension={extension} withPreloader setHeaderNodes={setHeaderNodes} />
       </PageSection>
-    </PageLayout>
+    </>
```

Comment:
As our core should support both plugin types (with the new design that you've implemented) and the old one, please make this feature switchable.
We need to use old Admin page structure for old plugins and the new one for the newest plugins.